internal/uvm/create_lcow.go

```diff
@@ -78,7 +79,7 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		},
 		BootFilesPath:         filepath.Join(os.Getenv("ProgramFiles"), "Linux Containers"),
 		KernelFile:            "kernel",
-		KernelDirect:          osversion.Get().Build >= 18286, // Use KernelDirect boot by default on all builds that support it.
+		KernelDirect:          kernelDirectSupported, // Use KernelDirect boot by default on all builds that support it.
```

Comment:
The comment here doesn't make much sense anymore, I'd reword it and move it up to where we set ```kernelDirectSupported```.